client/client.go

```diff
@@ -1103,6 +1145,7 @@ func (c *client) ScanRegions(ctx context.Context, key, endKey []byte, limit int,
 		if protoClient == nil {
 			return nil, errs.ErrClientGetProtoClient
 		}
+		//lint:ignore staticcheck
```

Comment:
maybe can use `//nolint:staticcheck`

Reply:
let me try, the golang ci lint cannot run in my laptop.

Reply:
I have run on my laptop, you need to change this code snippet with `nolint:staticcheck` as well :)

https://github.com/tikv/pd/blob/7e18a69734f2ae391516625ec0230c05b363f5c5/tools/pd-api-bench/cases/cases.go#L355